Firestore/Source/Local/FSTLevelDBQueryCache.h

```diff
@@ -32,9 +33,16 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  * Retrieves the global singleton metadata row from the given database, if it exists.
+ * TODO(gsoltis): remove this method once fully ported to transactions.
  */
 + (nullable FSTPBTargetGlobal *)readTargetMetadataFromDB:(std::shared_ptr<leveldb::DB>)db;
 
+/**
+ * Retrieves the global singleton metadata row using the given transaction, if it exists.
+ */
++ (nullable FSTPBTargetGlobal *)readTargetMetadataFromTransaction:
```

Comment:
It's odd but I think "from" makes sense as a preposition for the DB, but doesn't work for the transaction. We're reading from the database, but that's now implicit in the transaction. We're not reading "from" the transaction though. I think "with" makes more sense (and lines up with my suggestions above).

Reply:
Done.